modules/react/text-input/lib/InputGroup.tsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 
 import {space} from '@workday/canvas-kit-react/tokens';
-import {createComponent, ExtractProps, useIsRTL} from '@workday/canvas-kit-react/common';
+import {
+  createComponent,
+  dispatchInputEvent,
+  ExtractProps,
+  mergeProps,
+  useForkRef,
+  useIsRTL,
+} from '@workday/canvas-kit-react/common';
 import {Flex} from '@workday/canvas-kit-react/layout';
+import {TertiaryButton} from '@workday/canvas-kit-react/button';
+import {xSmallIcon} from '@workday/canvas-system-icons-web';
 
 import {TextInput} from './TextInput';
 
+const InputGroupContext = React.createContext<React.RefObject<HTMLInputElement>>({
+  current: null,
+});
+
```

Comment:
I'm sure this is not how you intended it to be used, but if you created a model, and have `inputGroup` use `createContainer` would that be another approach and that ways all it's sup components can uuse `createSubComponent` and you'd have access to the model?

Reply:
That is a good point. I could use a model. I was thinking about model pollution. It isn't a problem in this case, and we have ways around it.